docs/user-documentation/searching.md

```diff
@@ -0,0 +1,60 @@
+
+### SOLR Index Configuration
+
+The Default Solr content index user interface is divided into four tabs: 'View', 'Edit', 'Fields', and 'Processors'.
+
+#### View Tab
+
+The 'View' tab gives an overview of the index and its status:
+- 'Index status' is a progress bar that indicates how many items have been indexed.
+- 'Status' allows you to enable/disable the index from being used.
+- 'Datasource' indicates what types of entities are being indexed and how many have.
+- 'Tracker' states which item tracker is being used.
+- 'Server' provides a link to the SOLR server's configuration page.
+- 'Server index status' is a count of the total number of items returned from an unfiltered empty search. Generally speaking, this should match the total number of items indexed. See the [Drupal 7 Search API FAQ](https://www.drupal.org/node/2009804#server-index-status) for more details.
+- 'Cron batch size' displays how many items will be indexed when Drupal's cron runs.
+
+The 'View' tab also provides links to some common actions. 'Start Indexing Now' allows you to start a indexing job for a specified number of items (default is 'all'). You can also specify how many items should be indexed in each batch (default is '50'). The other links allow a repository manager to queue all objects for reindexing, clear the index, or rebuild tracking information.
+
+#### Edit Tab
+
+The 'Edit' tab allows repository managers to configure how the index works as a whole, including its name, the data sources—entity types—it can index (including which specific content types or taxonomies will be indexed), which server it is connected to, and other SOLR-specific options.
+
+'Content' (types) is the only data source enabled by default. Selecting 'Taxonomy term' will enable searching taxonomies which is recommended if the repository uses taxonomies for subjects or other discovery points. Once the data sources are enabled a configuration box for each of them will appear in a section just below the list of data sources. This allows repository managers to select which content types (or taxonomy vocabularies) will be included in the index. By default all the content types, and vocabularies if the taxonomy data source is enabled, are indexed.
+
+**Note:** the defaults assume a repository is adding content using the web interface. If a repository manager plans on bulk-loading content they should disable the 'Index items immediately' option in the expandable 'Index Options' box and increase the 'Cron batch size' option.
+
+#### Fields Tab
+
+The 'Fields' tab allows repository managers to select which fields will be indexed. The default set of fields enabled come from a standard Drupal installation and do not reflect the fields Islandora adds for 'Repository Item'. **Repository managers need to add the fields necessary for their Islandora instance.**
```

Comment:
Are there fields we could add by default that would lessen the burden of initial setup?

Reply:
@dannylamb I would assume this is possible, we simply haven't explored it yet. I figured document *as is* and come back later once someone takes the time to resolve it.

Reply:
I'm not suggesting we change the language here, so please don't hold anything up on my account.  I'll make an issue and we can try to find the scope of what defaults we can reasonably provide.  Time will tell most likely as more and more people go through the setup.